packages/clerk-js/src/core/resources/User.ts

```diff
@@ -146,6 +154,40 @@ export class User extends BaseResource implements UserResource {
     return new ExternalAccount(json, this.path() + '/external_accounts');
   };
 
+  generateTOTP = async (): Promise<TOTPResource> => {
+    const json = (
+      await BaseResource._fetch<ExternalAccountJSON>({
+        path: '/me/totp',
```

Comment:
@yourtallness The API is a bit different from the [design doc](https://www.notion.so/clerkdev/TOTP-Time-based-One-Time-Password-46320e464d024d938cc44c79468122da). It's `/me/totp` vs `/me/totp_secret`. Is this intentional?

Reply:
The ddoc has been updated with the new naming